samples/excel/85-preview-apis/events-workbook-activated.yaml

```
@@ -0,0 +1,80 @@
+order: 5
+id: excel-events-workbook-activated
+name: Workbook activated event
+description: This sample shows how to register a workbook activated event handler.
+host: EXCEL
+api_set:
+    ExcelAPI: '1.13'
+script:
+  content: |
+    $("#register-event-handler").click(() => tryCatch(registerEventHandler));
+
+    async function registerEventHandler() {
+      await Excel.run(async (context) => {
+        const workbook = context.workbook;
+
+        // Register the workbook activated event handler.
+        workbook.onActivated.add(workbookActivated);
+
+        await context.sync();
+        console.log("Added event handler for workbook activated.");
+      });
+    }
+
+    async function workbookActivated(event: Excel.WorkbookActivatedEventArgs) {
+      await Excel.run(async (context) => {
+        // Script that runs when the workbook is activated.
+        console.log("The workbook was activated.");
+      });
+    }
+
+    /** Default helper for invoking an action and handling errors. */
+    async function tryCatch(callback) {
+      try {
+        await callback();
+      } catch (error) {
+        // Note: In a production add-in, you'd want to notify the user through your add-in's UI.
+        console.error(error);
+      }
+    }
+  language: typescript
+template:
+    content: |-
+      <section class="ms-font-m">
+        <p>This sample shows how to register a workbook activated event handler.</p>
+        <p>Once the event handler is registered, a notification will print to the console when the workbook is activated. Try
+          switching to another application and then switching back to Excel to see the console notification.</p>
```

Nit: Avoid "will" for brevity.
```suggestion
        <p>Once the event handler is registered, a notification prints to the console when the workbook is activated. Try
          switching to another application and then switching back to Excel to see the console notification.</p>
```